README.md

```diff
@@ -1,38 +1,46 @@
-# Movies Browser
+# **Movies Browser** 🎥
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<p align="center"><img src="https://raw.githubusercontent.com/olachrzan/movies-browser/Icon%26OG-Image/public/app-screenshot.jpg" alt="App layout screenshot"></p>
 
```

Comment:
We can think about adding table of contents so that everyone can quickly find interesting content  😃